topology_test.py

```diff
@@ -443,6 +443,8 @@ def test_crash_during_decommission(self):
         cluster.populate(3).start()
 
         node1, node2 = cluster.nodelist()[0:2]
+        session = self.patient_cql_connection(node1)
+        session.execute("ALTER KEYSPACE system_distributed WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':'2'};")
```

Comment:
this is in other test methods too, without this, we are actually not decommissioning anything because decommission will fail as there will not be enough replicas if replication factor will be 3 and we want to decommission to have just 2. We need to lower this to 2 so by decommissioning we will have still enough.